### PR TITLE
[workers] document waf event in limits

### LINKED ISSUE #4764: [Workers] Request for clarification of anti-abuse protection
### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

- https://developers.cloudflare.com/workers/platform/limits/#request

### Section that requires update

Request

### What needs to change?

The `Request` section does currently not mention that you'll see firewall entries with `worker` as rule ID when the Workers anti-abuse protection has kicked in.
Currently customers may be quite confused when they suddenly see a lot of `worker` firewall entries, as they won't know that it's caused by the anti-abuse protection.

Here's an recent example of a customer who suddenly saw a lot of `worker` firewall entries, but wasn't aware that it was caused by the anti-abuse protection:
Screenshot 2022-06-09 at 21 18 11

Here's the response I gave for reference:
Screenshot 2022-06-09 at 21 20 27

### How should it change?

The `Request` section should include that the internal `worker` firewall rule is used by the Workers anti-abuse protection. Currently the `worker` firewall rule is nowhere mentioned in the documentation, so customers running into it may be a bit confused and not know what to do about it.

### Additional information

Not necessarily related to the documentation, but I would suggest Cloudflare to implement some kind of automatic notification when the anti-abuse protection has kicked in for a customer.
If for example, a customer is using Workers a lot internally for mission-critical work, it would help a lot if Cloudflare immediately notified them of traffic being too high and that they should immediately contact Cloudflare for a limit raise. Currently, Cloudflare will just drop a lot of requests without any notification, leaving customers completely clueless.

### PR DESCRIPTION
**Pull request by helloimalastair**
#4764

### CODE REVIEW

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ The burst rate and daily request limits apply at the account level, meaning that
 
 Accounts using the Workers Free plan are subject to a burst rate limit of 1000 requests per minute. Users visiting a rate limited site will receive a Cloudflare `1015` error page. However if you are calling your Worker programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code `429`.
 
+Workers being rate-limited by Anti-Abuse Protection are also visible from the Cloudflare Dashboard, in the form of a Web Application Firewall Block event, with a `ruleID` of `worker`.
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
Workers being rate-limited by Anti-Abuse Protection are also visible from the Cloudflare dashboard. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) > select your site > **Security** > **Overview** > scroll to **Activity log**  and review the log for a Web Application Firewall Block event with a `ruleID` of `worker`.
```

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -151,6 +151,8 @@ The burst rate and daily request limits apply at the account level, meaning that
 
 Accounts using the Workers Free plan are subject to a burst rate limit of 1000 requests per minute. Users visiting a rate limited site will receive a Cloudflare `1015` error page. However if you are calling your Worker programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code `429`.
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
Accounts using the Workers Free plan are subject to a burst rate limit of 1,000 requests per minute. Users visiting a rate limited site will receive a Cloudflare `1015` error page. However if you are calling your Worker programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code `429`.
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by arunesh90** [comment]
Bit late with this comment, but I think a note should be added to it that Cloudflare should be contacted to increase the limit of that anti-abuse protection.

**Comment by helloimalastair** [comment]
@arunesh90 does that apply to the free plan too?

**Pull request merged**